docs: iframe the future work widget (#537)
thousandbrainsproject/tbp.monty: docs/future-work/project-roadmap.md

![Screenshot of our overview sheet. For an up-to-date view, just go to the sheet https://docs.google.com/spreadsheets/d/1HlKqhMzYWxpW6_D0LtDd5qEuWRYHl6xMfxbIZ76lzng/edit?usp=sharing](../figures/future-work/monty_tasks.png)

Here is an interactive view of that table, that can be filtered and sorted to find tasks that match your scope appetite, skills and interests.f

[block:embed]
{
  "html": false,
  "url": "https://futureworkwidget.thousandbrains.ai/index.html?columns=title,estimated-scope,rfc,status,skills",
  "provider": "futureworkwidget.thousandbrains.ai",
  "href": "https://futureworkwidget.thousandbrains.ai/index.html?columns=title,estimated-scope,rfc,status,skills",
  "typeOfEmbed": "iframe",
  "height": "800px",
  "iframe": true
}
[/block]

These are tasks where we have a rough idea of how we want to achieve them but we haven't necessarily scheduled when we will work on them or who will work on them. **We welcome anyone who would like to pick up one of these tasks and contribute to it.**

Tasks are categorized in two ways:

> - **Which part of Monty does it touch?** This is represented in the columns, and is organized into Sensor Module, Learning Module, Motor System, Voting, Hierarchy, Environment, Framework, and Open-Source/Communication.

Reviewer: Is this still referring to the spreadsheet? If not, I think the wording needs to be updated (e.g. "This is represented in the columns,..." isn't accurate for the interactive table. Why was the first bullet point removed? Don't we still capture this with the "Metric" field?

Tasks that are done have a completed next to them and are shaded in green. When a task gets checked off, it will add progress to the corresponding capabilities on the right.

# What the TBP Team is Working On

> Some of the tasks are under active development by our team or scheduled to be tackled by us soon. Those have a GitHub avatar icon in the status column.

Reviewer: ```suggestion
Some of the tasks are under active development by our team or external contributors. Those have a GitHub avatar icon in the status column.
```
